### ISSUE TITLE
Handle gettimeofday gracefully

### ISSUE THREAD

**Issue by dol** [vimeo/psalm#871]
### Summary of a problem or a feature request

[gettimeofday](https://secure.php.net/manual/en/function.gettimeofday.php) has  a second param when set to true will return a float.
phpstan is complaining about a binary operation of a array|float value. See code snipper for more details.

### Code snippet that reproduces the problem

```php
<?php declare(strict_types = 1);

gettimeofday(true) - gettimeofday(true);
```

Output:
```
ERROR: PossiblyInvalidOperand - src/test.php:3:1 - Cannot add an array to a non-array float
gettimeofday(true) - gettimeofday(true);
```
### Expected output

No warning.

**Comment by muglug** [comment]
Thanks!
